docs/formulary/densities.rst

```diff
@@ -0,0 +1,9 @@
+.. _densities:
+
+******************************************************************
+Density related functionality (`plasmapy.formulary.densities`)
```

Comment:
To match the others, I think this should say 
"Density Plasma Parameters" 

e.g. look at frequencies.rst